spring-ai-alibaba-graph/spring-ai-alibaba-graph-studio/graph-ui/src/pages/Graph/Design/types/EndNode.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon } from '@iconify/react';
+import { Handle, Position } from '@xyflow/react';
+import { Flex } from 'antd';
+import React from 'react';
+import './base.less';
+
+type props = {
```

Comment:
This is fair. I used the "any" type in quite a few places in this PR. It might take some time to fully refine them later, especially for the "data" field，and there is a great deal of uncertainty in the DSL.